sw/device/tests/sim_dv/sleep_pin_retention_test.c

```diff
@@ -105,6 +107,15 @@ void gpio_test(dif_pwrmgr_t *pwrmgr, dif_gpio_t *gpio, int round) {
   //     Chosen GPIO value: %2x
   LOG_INFO("Chosen GPIO value: %2x", gpio_val);
 
+  // 4. Configure PINMUX Retention value opposite to the chosen value.
+  pad_kind = kDifPinmuxPadKindMio;
```

Comment:
i cannot remember now, but do we in this test also check the case where we want to "persist" the gpio value?

Reply:
persist (passthrough) is to be tested in pwm as far as I know. Other values for entire PADs are checked in `chip_sw_sleep_pin_mio_dio_val`.